ci/Jenkinsfile_docker_cache

```diff
@@ -38,6 +38,7 @@ core_logic: {
         timeout(time: total_timeout, unit: 'MINUTES') {
           utils.init_git()
           sh "python3 ./ci/docker_cache.py --docker-registry ${env.DOCKER_CACHE_REGISTRY}"
+          sh "cd ci && docker-compose -f docker/docker-compose.yml build --parallel && docker-compose -f docker/docker-compose.yml push "
```

Comment:
Can you pull that into docker_cache.py to leverage parallelisation wrt non updated images and also providing one single interface?

Reply:
The idea is to delete `docker_cache.py` later, so the `docker-compose -f docker/docker-compose.yml build --parallel && docker-compose -f docker/docker-compose.yml push` would be the unified interface.

Reply:
I'm not sure what parallelism you refer to. Is `docker-compose build --parallel` not sufficient?

Reply:
I understand. 

It's not sufficient in the sense of that docker-cache and docker compose are run in sequence. Alternatively, create a second job so it runs on another node?

Reply:
`docker-cache` will only be used temporarily. I think the current solution is fine given that the first step (`sh "python3 ./ci/docker_cache.py`) will be deleted later.